--- a/amd/backport/kcl_mmu_notifier.c	2020-05-07 23:08:40.169682825 -0400
+++ b/amd/backport/kcl_mmu_notifier.c	2020-05-07 23:08:51.826756037 -0400
@@ -1,6 +1,6 @@
 #include <kcl/kcl_sched_mm_h.h>
 
-#if !defined(HAVE_MMU_NOTIFIER_CALL_SRCU)
+#if !defined(HAVE_MMU_NOTIFIER_CALL_SRCU) && !defined(HAVE_MMU_NOTIFIER_PUT)
 void mmu_notifier_unregister_no_release(struct mmu_notifier *mn,
 					struct mm_struct *mm)
 {
